src/main/java/com/google/edith/DealItem.java

```diff
@@ -91,6 +95,44 @@ public void setComment(String comment) {
     this.comment = comment;
   }
 
+  /**
+   * Input string expiration is expected to have the minimum expiration time, the maximum expiration
+   * time, and the unit of time, in no particular order. It parses the string to find the minimum
+   * expiration time and the unit, as that is the data needed to generate new grocery lists based
+   * off of.
+   *
+   * <p>ex: '1.0 2.0 Weeks' -> this.expiration = '1.0 Weeks'
```

Comment:
Should the input always expect to include weeks as unit? Or could it be days or months?

Reply:
can be days and months too, I'll include that in the javadoc!